backend/telemetry_shard/src/aggregator.rs

```diff
@@ -247,6 +251,22 @@ impl Aggregator {
                         .send_async(FromShardAggregator::UpdateNode { local_id, payload })
                         .await;
                 }
+                ToAggregator::FromWebsocket(conn_id, FromWebsocket::Remove { message_id }) => {
+                    // Get the local ID, ignoring the message if none match:
+                    let local_id = match to_local_id.get_id(&(conn_id, message_id)) {
+                        Some(id) => id,
+                        None => continue,
+                    };
+
```

Comment:
Do you think we should collect stats on the nr of ignored messages?

Reply:
We do collect stats on the number of messages we ignore because there are too many to deal with at the mo. This code just silently ignores messages from Ids that don't exist incase nodes are removed while messages that work on them are still queued for some reason (I'm not sure if I'd ever actually expect to encounter the "None" case though)